src/tests/unit/tests/views/page/__snapshots__/page.test.tsx.snap

```diff
@@ -1,19 +1,14 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`page view renders 1`] = `
-<React.Fragment>
-  <NarrowModeDetector
-    Component={[Function]}
-    childrenProps={
-      {
-        "deps": {},
-      }
-    }
-    deps={{}}
-    isNarrowModeEnabled={true}
+<DocumentFragment>
+  <mock-narrowmodedetector
+    childrenprops="[object Object]"
+    deps="[object Object]"
+    isnarrowmodeenabled="true"
```

Comment:
instead of as fragment can we use 
expectMockedComponentPropsToMatchSnapshots(NarrowModeDetector) so that we can match close to old snapshot

Reply:
Yes, implemented suggested changes.